internal/simplename/simplename.go

```diff
@@ -0,0 +1,31 @@
+package simplename
+
+import (
+	"errors"
+)
+
+var ErrNotASimpleName = errors.New("name contains restricted characters, please only use [A-Za-z0-9-_.]")
```

Comment:
Should we allow `:` similar to how we have tags for images?

Reply:
Added in https://github.com/cirruslabs/orchard/pull/129/commits/54bf2d005e41ee6f0a7213b55f642e55430021a5.